sdk/areas/areas.go

```
@@ -96,12 +97,12 @@ func (c *Client) GetArea(ctx context.Context, userAuthToken, serviceAuthToken, c
 	if next, ok := body["next"]; ok && (serviceAuthToken != "" || userAuthToken != "") {
 		b, err = json.Marshal(next)
 		if err != nil {
-			return
+			return areaDetails, nil
```

Swallowing the error.

---

Fixed in update 1.